core/templates/components/state-directives/outcome-editor/outcome-editor.component.html

```diff
@@ -153,6 +153,61 @@
   </div>
 </div>
 
+<br *ngIf="displayFeedback">
+
+<div *ngIf="!destinationIfStuckEditorIsOpen && !isInQuestionMode() && destIfStuckFeatEnabled"
```

Comment:
```suggestion
<div *ngIf="destIfStuckFeatEnabled && !destinationIfStuckEditorIsOpen && !isInQuestionMode()"
```
Better because of lazy evaluation.

Reply:
Done.